src/store/metadata/stub/store.py

```diff
@@ -178,15 +178,21 @@ def get_sub_topics(self, topic_id: str) -> Dict:
         if topic is None:
             return None
 
-        sub_topic_ids = topic["sub_topics"]
         topics = self.get_topics()
-        sub_topics = [x for x in topics if x["@id"] in sub_topic_ids]
 
-        if sub_topics is None:
+        sub_topics = [
+            x
+            for x in topics["topics"]
+            if len(x["parent_topics"]) > 0 and topic["@id"] in x["parent_topics"]
+        ]
+
+        if len(sub_topics) == 0:
             return None
         return contextualise(
             {
-                "items": sub_topics,
+                "@id": "https://staging.idpd.uk/topics",
```

Comment:
well done, but you'll need to update the id too, i.e `@id: "http://localhost:8000/topics/{topic_id}/subtopics",`